java/java.source/nbproject/org-netbeans-modules-java-source.sig

```diff
@@ -1265,6 +1267,7 @@ meth public static javax.lang.model.type.WildcardType resolveCapturedType(javax.
 meth public static org.netbeans.api.lexer.TokenSequence<org.netbeans.api.java.lexer.JavaTokenId> getJavaTokenSequence(org.netbeans.api.lexer.TokenHierarchy,int)
 meth public static org.openide.filesystems.FileObject getFile(javax.lang.model.element.Element,org.netbeans.api.java.source.ClasspathInfo)
 meth public static org.openide.filesystems.FileObject getFile(org.netbeans.api.java.source.ElementHandle<? extends javax.lang.model.element.Element>,org.netbeans.api.java.source.ClasspathInfo)
+meth public static void forceSource(org.netbeans.api.java.source.CompilationController,org.openide.filesystems.FileObject)
```

Comment:
Something @jlahoda shall check...

Reply:
I've added this method recently (to aid Java refactoring and similar usecases while running without nb-javac).